lib/reel/session/crypto.rb

```diff
@@ -14,10 +14,10 @@ def encrypt val
         cipher = OpenSSL::Cipher::AES128.new :CBC
         cipher.encrypt
         # getting config depending on call from session/store
-        config = Reel::Session.configuration
+        config = @config || self.session_config
```

Comment:
`@config` will only already exist when this method is called on a `Store`, correct? so this is basically a "we need the session config, we're in a module that is mixed into either `Request` or `Store`, so get it from one of those places." I'm starting to think crypto methods maybe should be static, and not mixed into those classes. That refactor would require `Request` and `Store` to pass along the session_config obj with each enc/decrypt call. What do you think? It would keep these methods out of those classes. 🤔